mitodl/micromasters
# Add handling for next parameter

#### What are the relevant tickets?
Fixes #2385 

#### What's this PR do?
In the homepage the `next` query parameter is set in the link to login to edx. If the `next` query parameter is set for the link it is used as the redirect URI after the signin is completed. Since Django sets `?next` if the user attempts to go somewhere that requires a login, this should allow the user to just click the login link to get back where they were, including all relevant query parameters.

#### How should this be manually tested?
Log out, then go to `/dashboard/?xy=z`. You should be redirected to the home page. Click the login link to log in, then you should be redirected to edX and be directed back to end up at `/dashboard/?xy=z`.
